substrate/client/consensus/beefy/src/lib.rs

```diff
@@ -32,7 +32,7 @@ use crate::{
 	metrics::register_metrics,
 };
 use futures::{stream::Fuse, FutureExt, StreamExt};
-use log::{debug, error, info, warn};
+use log::{debug, error, info, log_enabled, trace, warn, Level};
```

Comment:
nit:

are `log_enabled` and `Level` used?

Reply:
Good catch, will remove